packages/documentation/src/stories/components/subnavigation/subnavigation.stories.ts

```diff
@@ -0,0 +1,94 @@
+import { Meta, StoryObj, Args, StoryContext } from '@storybook/web-components';
+import { BADGE } from '../../../../.storybook/constants';
+import { html } from 'lit';
+
+const meta: Meta = {
+  title: 'Components/Subnavigation',
+  parameters: {
+    badges: [BADGE.WEB_COMPONENT_CANDIDATE],
+  },
+  render: render,
+  argTypes: {
+    coloredBackground: {
+      name: 'Colored Background',
+      control: {
+        type: 'boolean',
+      },
+    },
+    alternate: {
```

Comment:
The `.subnavigation-alternate` class only set a background color therefore it is no longer relevant: users can use the `.bg-light` class instead.